File: 1 - BackEnd/Vanilla/PHP/3 - lang/3 - operadores.php
<?php

/* ##########============================########## */
/* ######===--- Operadores aritmeticos ---===###### */
/* ##########============================########## */

$operacion = (5 + 5);
$operacion = (5 - 5);
$operacion = (5 * 5);
$operacion = (5 / 5);
$operacion = (5 % 5);

// Operador (^/**): Representa la exposicion.
$operacion = (5 ** 5);
$operacion = (5 ^ 5);

// (.), representa la concatenacion.
$operacion = (5 . "5"); // "55"

/* ##########========================########## */
/* ######===--- Operadores logicos ---===###### */
/* ##########========================########## */

(5 == 5);
(5 < 5);
(5 > 5);
(5 <= 5);
(5 >= 5);

// Diferente a.
(5 != 5);
(5 <> 5);

// ----------------------------- //
// ------ Comparar tipado ------ //
// ----------------------------- //

// Comparamos no solo los datos, sino tambien su tipado.
(5 === 5);
(5 !== 5);
(5 <== 5);
(5 >== 5);

// ---------------------------------- //
// ------ Mas de una condicion ------ //
// ---------------------------------- //

(5 === 5 && 5 > 10);
// Operador (&&): Compara dos condiciones, las dos deben cumplirse.

(5 === 5 || 5 > 10);
// Operador (||): Compara dos condiciones, Solo una de ellas debe cumplirse.

(! (5 === 5))
// Operador (!): Convierte el resultado de la condicion a su contraparte.

(5 == "5") // true
(5 === "5") // false

File: 1 - BackEnd/Vanilla/Python/3 - lang/3 - operadores.py
# ##########============================##########
# ######===--- Operadores aritmeticos ---===######
# ##########============================##########

operacion = (5 + 5)
operacion = (5 - 5)
operacion = (5 * 5)
operacion = (5 / 5)
operacion = (5 % 5)

# Exponente.
operacion = (5 ** 5)

# Concatenación.

# Hay que convertir explicitamente antes de concatenar.
operacion = str(5) + "5"  # "55"
# TypeError: unsupported operand type(s) for +: 'int' and 'str'
operacion = 5 + "5"

# ##########========================##########
# ######===--- Operadores logicos ---===######
# ##########========================##########

(5 == 5)
(5 < 5)
(5 > 5)
(5 <= 5)
(5 >= 5)

# Diferente a.
(5 != 5)

# ----------------------------- #
# ------ Comparar tipado ------ #
# ----------------------------- #

# Comparamos solo los datos.
(5 == 5);
(5 != 5);
(5 <= 5);
(5 >= 5);

# ---------------------------------- #
# ------ Mas de una condicion ------ #
# ---------------------------------- #

# Operador and: Ambas condiciones deben ser True.
(5 == 5 and 5 > 10)

# Operador or: Al menos una condición debe ser True.
(5 == 5 or 5 > 10)

# Operador not: Invierte el resultado de la condición.
not (5 == 5)

(5 == "5") # False